benefits/eligibility/templates/eligibility/includes/selection-label--veteran.html

```diff
@@ -7,8 +7,9 @@
 
 {% block description %}
   {% blocktranslate trimmed %}
-    This option is for people who have served in the active military, naval, or air service, and who were discharged or
-    released therefrom under conditions other than dishonorable. You will need to
-    <a href="https://www.va.gov/resources/verifying-your-identity-on-vagov/" target="_blank" rel="noopener noreferrer">verify your identity through VA.gov</a>.
+    This option is for people who have served in the active military, naval, or air service, and who were discharged or released therefrom under conditions other than dishonorable. You will need to verify your identity with
   {% endblocktranslate %}
+
+  {% include "core/includes/modal-trigger.html" with classes="border-0 bg-transparent p-0 login" modal="modal--login-gov" login=True period=True %}
```

Comment:
@thekaveman @angela-tran This modal trigger link opens the `modal--login-gov` modal, which is included into the page from the `selection-label--senior.html` file. That means that technically, under this design, if a transit agency *ONLY* offers Veterans benefits, but doesn't offer an Older Adult one, this template will have to be modified to also include the modal code. In other words, the Veteran selection label would require the Older Adult one to also be there.

Alternatively, I could also add this line here:

```
{% include "eligibility/includes/modal--login-gov-help.html" with id="modal--login-gov" size="modal-lg" header="p-md-2 p-3" body="pb-md-3 mb-md-3 mx-md-3 py-0 pt-0 absolute-top" %}
```

but then this HTML modal code would be duplicated on this page, and since there would be 2 modals with the same id (`modal--login-gov`), we'd have to have 2 different IDs pointing to duplicated HTML. Not sure which approach is better.

Reply:
Hmm, that is tricky. With your second idea - the one about including the login-gov-help modal in the veteran selection-label - maybe the selection-label include templates can have a variable that says whether to include the modal or not?

Reply:
Hrmm - yeah, but that variable would have to live in `0002_data.py` because that's where the radio button selector templates get required for each agency: 
<img width="737" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/0befb429-3867-4c74-aaf5-d273ac836ee4">

And that seems like a little too much for this situation?

Might just be better to go with adding the modal _all_ the time just to make each selection template independent.

Reply:
Update: https://github.com/cal-itp/benefits/pull/1706/commits/ff4ee183ace750a19cbdab363386748be66758b1

The page now has 2 modals with 2 different IDs. The Veteran selector will have a modal with the ID `"modal--login-gov-veterans"` and the trigger will open that one.

Reply:
> And that seems like a little too much for this situation?
>
> Might just be better to go with adding the modal all the time just to make each selection template independent.

Oh I see. Yeah, I didn't look too deeply at how the variable would get passed in, and what you're saying makes sense. Seems like having 2 modals is our best option here